packages/ui/src/components/editor/input/Euler/index.tsx

```diff
@@ -53,48 +52,42 @@ const getBoundedRoundedAngle = (angle: number) => {
 const tempEuler = new Euler() // we need the persistance, the hookstate doesnt register the dynamically allocated euler and quat value otherwise, thus we cannot assign new variable to the same
 export const EulerInput = (props: EulerInputProps) => {
   tempEuler.setFromQuaternion(props.quaternion, 'YXZ')
-  const angle = useHookstate({
+  const angle = {
```

Comment:
this change doesnt make any sense, why is should this no longer be reactive?

Reply:
Setting the values directly wasnt actually being applied in the hookstate, setting them explicitly seemed to work, i checked if we needed the hookstate at all and after some testing it didnt seem necessary

Reply:
the component, itself is using the quat as prop, there are two cases
1. we use the gizmo to change quat
2. we use the scrubber to change euler

in case one 
the component gets rerendered rapidly as the prop changes, so we always have the fresh angle value and thus appropriate values in the input fields 
in case two 
the callback changes the quat causing a rerender, which in turn gives us a fresh angle value

in either case, the angle value itself does not need to be reactive   
@HexaField